bluemix/configuration/core_config/repository.go

```diff
@@ -46,6 +51,10 @@ type Repository interface {
 	UpdateCheckInterval() time.Duration
 	UpdateRetryCheckInterval() time.Duration
 	UpdateNotificationInterval() time.Duration
+	// VPCCRITokenURL() returns the value specified by the environment variable 'IBMCLOUD_CR_VPC_URL', if set.
+	// Otherwise, the default VPC auth url "http://169.254.169.254" is returned
```

Comment:
Rather than including this URL in the comment, this should just point to the constant. Just in case this value ever changes for some reason, it is only updated in one place, then.